app/addons/documents/views.js

```diff
@@ -505,21 +517,37 @@ function(app, FauxtonAPI, Components, Documents, Databases, Views, QueryOptions)
     },
 
     beforeRender: function() {
-      var docs;
-
       this.removeNestedViews();
 
       this.pagination.setCollection(this.collection);
       this.allDocsNumber.setCollection(this.collection);
 
-      docs = this.expandDocs ? this.collection : this.collection.simple();
```

Comment:
No it's no longer needed. It used to handle showing a subset of the data (id + rev only). Now nothing is shown - just the ID row.